001-cron.php

```diff
@@ -16,6 +16,11 @@
  * @return bool
  */
 function wpcom_vip_use_core_cron() {
+
+	if ( defined( 'WPCOM_VIP_USE_CORE_CRON' ) && WPCOM_VIP_USE_CORE_CRON ) {
```

Comment:
We've started to drop the `WPCOM_` prefix. Plus, add a disclaimer.

```suggestion
	// For some applications, Cron Control is not a good fit. Allow those apps to use core cron instead. 
	// Note: please check with the VIP team before using this constant as it's not always guaranteed to work.
	if ( defined( 'VIP_USE_CORE_CRON' ) && VIP_USE_CORE_CRON ) {
```